src/pages/index.js

```diff
@@ -62,21 +63,21 @@ const features = [
     topTitle: "Student Space",
     title: "Your space to learn",
     description:
-      "Lorem ipsum dolor sit amet consectetur adipisicing elit. Saepe eos, itaque perferendis eaque iure sequi cumque ea nostrum corrupti maiores fugit culpa praesentium velit quis deserunt distinctio natus atque dolore?",
+      "Customize it, get access to the courses based on your interest, and plan your learning path. The student space is the perfect place to improve the way that you learn while you follow your progress and get in touch with the instructor and other students.",
     image: <img src="img/img01.svg" alt="Img 01" />,
   },
   {
     topTitle: "Instructor Space",
     title: "Your space to learn",
     description:
-      "Lorem ipsum dolor sit amet consectetur adipisicing elit. Saepe eos, itaque perferendis eaque iure sequi cumque ea nostrum corrupti maiores fugit culpa praesentium velit quis deserunt distinctio natus atque dolore?",
+      "The control of the content in your hands. Get statistics about the student progress, and course statistics. Find out how to improve the way that your content is delivered to the students while you plan your course or next class.",
```

Comment:
`The control of the content in your hands` or `The control of the content is in your hands`?